packages/docs-site/src/library/pages/components/masthead.md

```diff
@@ -107,13 +105,13 @@ The masthead is a simple header section for a website that
     Name: 'homeLink',
     Type: 'LinkTypes',
     Required: 'True',
-    Description: 'An object typically containing a to or href property to indicate the property to use in the LinkComponent to send the user back to the homepage for the service.',
+    Description: 'An object typically containing a to or href property to indicate the property LinkComponent to send the user back to the homepage for the service.',
```

Comment:
Could there be some back ticks around "to" and "href"?

Reply:
We can add back ticks, however they won't render a code block inside the table due to the limitation of the table implementation. This should be fixed when we implement the new, proposed table designs.